std/stdio.d

```diff
@@ -2226,6 +2298,42 @@ $(XREF file,readText)
         assert(blc.front is blc.front);
     }
 
+    /**
+    Parse from the file handle one line at a time into a record tuple.
```

Comment:
Oh thanks for pointing out my mistake.
I saw you got rid of "one line at a time" - imho this is important to make it obvious its line by line.